minimisers/NSMain/split_ellipsoid.m

```diff
@@ -174,5 +174,12 @@
 
 if DEBUG; fprintf('SPLIT ELLIPSOID: min F(S) = %f\n', minFS); end;
 
-return
+end
+
+function x = makeCell(n,vals)
```

Comment:
The exact function exist in `refPrctileConfInts.m` so I would unify

Reply:
Ok